hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/EdgesAPI.java

```diff
@@ -112,7 +112,7 @@ public String scan(@Context GraphManager manager,
         query.scan(start, end);
         query.page(page);
         if (query.paging()) {
-            query.limit(SCAN_LIMIT);
+            query.limit(SCAN_PAGE_LIMIT);
```

Comment:
add page_limit parameter and set default value as SCAN_PAGE_LIMIT
rename SCAN_PAGE_LIMIT to SCAN_DEFAULT_PAGE_LIMIT